server/tests.go

```diff
@@ -16,9 +19,44 @@ func (ge *TestsEndpoint) AttachHandlers(r *mux.Router) {
 	r.HandleFunc("/ping", ge.ping)
 }
 
+func (ge *TestsEndpoint) checkExternal() bool {
+	isHealthy := true
+
+	if remote_signer.EnableRethinkSKS {
+		conn := database.GetConnection()
+
+		if !conn.IsConnected() {
+			isHealthy = false
```

Comment:
We should log the reason for which a health check fails

Reply:
+1